android/app/src/main/java/app/covidshield/module/CovidShieldModule.kt

```diff
@@ -28,9 +28,13 @@ class CovidShieldModule(context: ReactApplicationContext) : ReactContextBaseJava
     @ReactMethod
     fun getRandomBytes(size: Int, promise: Promise) {
         promise.launch(this) {
-            val bytes = SecureRandom().generateSeed(size)
-            val base64Encoded = Base64.encodeToString(bytes, Base64.DEFAULT)
-            promise.resolve(base64Encoded)
+            try {
```

Comment:
There is no need to try / catch here because `.launch` extension already handle exception.

Maybe next time, please ping me to PR for review whenever we change native code as I would like to bring it over to original repo too. 🙇 

cc @jeberhardt @timarney